ajax/libs/backbone.stickit/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "backbone.stickit",
-  "description": "Model binding in Backbone style.",
   "filename": "backbone.stickit.min.js",
+  "description": "Model binding in Backbone style.",
```

Comment:
Uh... I just want to make name and filename be the first and second line, so if it's unnecessary, I can modify it back.